docs/core/porting/upgrade-assistant-install.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to install the .NET Upgrade Assistant
+description: This article describes how to install the .NET Upgrade Assistant tool, which can be installed as a Visual Studio extension or a dotnet command-line tool.
+author: adegeo
+topic: how-to
+ms.date: 05/16/2023
+---
+# Install the .NET Upgrade Assistant
+
+The .NET Upgrade Assistant can be installed as a Visual Studio extension or as a .NET command-line tool. When installed as a Visual Studio extension, loaded projects can be upgraded through the context menu. The .NET command-line tool version of the tool provides an interactive step-by-step expreience. For more information about the tool, see [Overview of the .NET Upgrade Assistant](upgrade-assistant-overview.md).
+
+### Prerequisites
+
+- Windows Operating System
+- [Visual Studio 2022 version 17.1 or later](https://visualstudio.microsoft.com/downloads/)
+- [.NET 6 SDK](https://dotnet.microsoft.com/download/dotnet/6.0), for the command-line tool
```

Comment:
I'm not sure.. The original tool required .NET 6 but they're releasing a whole new version at build, so I think this will change. But they want these (the VS part) ready for build.